src/main/java/org/commonjava/util/sidecar/services/ProxyService.java

```diff
@@ -213,12 +221,12 @@ private String getTraceId( MultiMap headers )
     }
 
     @FunctionalInterface
-    private interface CheckedFunction<T, R>
+    private interface Function<T, R>
```

Comment:
This is important, not just a name change, only Function<T,R> works in GraalVM compilation